src/user.ts

```diff
@@ -215,4 +216,17 @@ export class UserModule {
             .then((r: any) => r.getData())
             .then((result: any) => result.bootstrapResource.current.featureFlags);
     }
+
+    /**
+     * Initiates SPI SAML SSO
```

Comment:
Can you please add a JSDOC comment for the `relayState` parameter? The comment should be something like

```ts
/**
 * Initiates SPI SAML SSO
 * @param relayState URL of the page where the user is redirected after a successful login.
 */
```